src/main/js/constants/constants.js

```diff
@@ -0,0 +1,21 @@
+
+const CONSTANTS = {
```

Comment:
Deliberately left this out of html5-common since only MainHtml5 should be concerned with these values